.azdo/pipelines/azure-dev.yml

```
@@ -4,9 +4,14 @@ trigger:
 pool:
   vmImage: ubuntu-latest
 
-container: mcr.microsoft.com/azure-dev-cli-apps:latest
-
 steps:
+
```

```suggestion
steps:
   - name: Checkout
```

Remove redundant newline